server/testfixtures/test-repos/automerge/exp-output-auto-policy-check.txt

```diff
@@ -0,0 +1 @@
+no template matched–this is a bug
```

Comment:
what's going on here? lol

Reply:
this is the error that you get if you don't have a template specified for the command :D I didn't want to add a template into this PR so I this is a placeholder.